update-sdk.sh

```diff
@@ -2,4 +2,4 @@
 
 echo "updating github.com/common-fate/sdk to $1"
 
-go get github.com/common-fate/sdk@$1
\ No newline at end of file
+go get github.com/common-fate/sdk@0f7f602b15f7a5e448b59e42f5182c91ed669e98
```

Comment:
this looks like it was accidentally committed from your dev environment, you can revert this

Reply:
Oh yes you're right, I'll change it back to $1